app/templates/gentelella/users/events/index.html

```diff
@@ -90,48 +90,5 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
-    <script type="text/javascript">
```

Comment:
I don't see why this is even here. As you can see [here](https://github.com/fossasia/open-event-orga-server/blob/development/app/templates/gentelella/users/events/index.html) .. this section doesn't even exist in the current `dev`. So, we don't want a commit that removes something that isn't there. 

I suggest you clean-up your branch and make a fresh commit on top of the latest `dev`.